premake5.lua

```diff
@@ -247,6 +247,9 @@
 		filter "system:macosx"
 			defines     { "LUA_USE_MACOSX" }
 			links       { "CoreServices.framework", "Foundation.framework", "Security.framework", "readline" }
+			
+		filter "system:linux or macosx"
```

Comment:
Why `or macosx`?

Reply:
Sorry for the late reply. And that would be something I forgot to takeout while messing with it, will fix in a sec

Reply:
Should be fixed now